src/routes/auth/ic-connect/+page.ts

```diff
@@ -11,9 +11,12 @@ export const ssr = false;
 export const load: PageLoad = async ({ url, parent }) => {
   await parent();
 
+  const forceLogin: boolean =
+    url.searchParams.get("forceLogin") === "1" || false;
```

Comment:
ça se discute, mais d'habitude on a plutot des trucs de la forme word1_word2; (referrer_uri, login_hint). on a aussi du word1-word2, mais je pense qu'il vaut mieux qu'on standardise sur word1_word2.
```suggestion
    url.searchParams.get("force_login") === "1" || false;
```